### ISSUE TITLE
-DBUILD_PARALLEL=ON fails on mac (at least)

### ISSUE THREAD

**Issue by doutriaux1** [UV-CDAT/uvcdat#667]
maybe as simple as a tweak to setup_runtime.[c]sh
```
[dash:~/ACME-data] mccoy20% /Users/mccoy20/uvcdat-app/14-09-16/bin/python gen_pngs.py

Traceback (most recent call last):

  File "gen_pngs.py", line 1, in <module>

    import vcs, cdms2, os, sys

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/__init__.py", line 31, in <module>

    from utils import *

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/utils.py", line 6, in <module>

    import boxfill

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/boxfill.py", line 30, in <module>

    import Canvas

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/Canvas.py", line 35, in <module>

    from queries import *

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/queries.py", line 31, in <module>

    import texttable, textorientation, textcombined, template, dv3d

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/vcs/dv3d.py", line 15, in <module>

    from DV3D.ConfigurationFunctions import ConfigManager

  File "/Users/mccoy20/uvcdat-app/14-09-16/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/DV3D/ConfigurationFunctions.py", line 6, in <module>

    import sys, vtk, cdms2, traceback, os, cdtime, cPickle, copy 

  File "/Users/mccoy20/uvcdat-app/14-09-16/Externals/lib/paraview-4.1/site-packages/vtk/__init__.py", line 85, in <module>

    from vtkIONetCDF import *

  File "/Users/mccoy20/uvcdat-app/14-09-16/Externals/lib/paraview-4.1/site-packages/vtk/vtkIONetCDF.py", line 1, in <module>

    from vtkIONetCDFPython import *

ImportError: dlopen(/Users/mccoy20/uvcdat-app/14-09-16/Externals/lib/paraview-4.1/site-packages/vtk/vtkIONetCDFPython.so, 2): Symbol not found: __ZNSt8ios_base4InitD1Ev

  Referenced from: /Users/mccoy20/uvcdat-app/14-09-16/Externals/lib/libnetcdf_c++.4.dylib

  Expected in: flat namespace

 in /Users/mccoy20/uvcdat-app/14-09-16/Externals/lib/libnetcdf_c++.4.dylib
```

**Comment by aashish24** [comment]
@doutriaux1 is this still an issue?

**Comment by doutriaux1** [comment]
I believe so. But let me try it again this morning.

**Comment by doutriaux1** [comment]
nope still here on my mac 10.8
Something in ParaView. So BUILD_PARALLEL=ON triggers ParaView, is that really what we want by the way?

**Comment by doutriaux1** [comment]
here's what I get:
```

  Linking C static library ../../../lib/libvtkWrappingTools-pv4.1.a

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/qr.cpp.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/RegularExpression.cxx.o

  [ 0%] Building C object
  VTK/ThirdParty/tiff/vtktiff/CMakeFiles/vtkmkg3states.dir/mkg3states.c.o

  [ 0%] Building C object VTK/ThirdParty/jpeg/vtkjpeg/CMakeFiles

  -------------------------------------------------

   From /Users/doutriaux1/build/ParaView-prefix/src/ParaView-stamp/ParaView-build-err.log

  -------------------------------------------------

  
  /Users/doutriaux1/build/build/ParaView/ThirdParty/IceT/vtkicet/src/strategies/radixk.c:24:10:
  fatal error: 'mpi.h' file not found

  #include <mpi.h>

           ^

  1 error generated.

  make[5]: ***
  [ThirdParty/IceT/vtkicet/src/ice-t/CMakeFiles/IceTCore.dir/__/strategies/radixk.c.o]
  Error 1

  make[4]: ***
  [ThirdParty/IceT/vtkicet/src/ice-t/CMakeFiles/IceTCore.dir/all] Error 2

  make[4]: *** Waiting for unfinished jobs....

  make[3]: *** [all] Error 2

  -------------------------------------------------



make[2]: *** [ParaView-prefix/src/ParaView-stamp/ParaView-build] Error 1
make[1]: *** [CMakeFiles/ParaView.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by jbeezley** [comment]
Also mentioned here: #789.  For one, I would recommend trying with `make -j1`.

It looks to me like IceT is using `MPI_C_INCLUDE_PATH` like everything else to find the header.  I'll try an MPI build to see if I run into the same issue.

**Comment by doutriaux1** [comment]
ok will try the make -j1, but if that works then we need to figure out which dependencies are wrong.

**Comment by doutriaux1** [comment]
make -j1 output
```
[ 41%] Built target g2clib
Scanning dependencies of target MPI
[ 42%] Creating directories for 'MPI'
[ 42%] Performing download step (download, verify and extract) for 'MPI'
-- downloading...
     src='http://uv-cdat.llnl.gov/cdat/resources/openmpi-1.6.4.tar.gz'
     dst='/Users/doutriaux1/build/openmpi-1.6.4.tar.gz'
     timeout='none'
attempt 1
-- [download 0% complete]
-- [download 100% complete]
-- downloading... done
-- verifying file...
     file='/Users/doutriaux1/build/openmpi-1.6.4.tar.gz'
-- verifying file... done
-- extracting...
     src='/Users/doutriaux1/build/openmpi-1.6.4.tar.gz'
     dst='/Users/doutriaux1/build/build/MPI'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[ 43%] No patch step for 'MPI'
[ 43%] No update step for 'MPI'
[ 43%] Performing configure step for 'MPI'
-- MPI configure command succeeded.  See also /Users/doutriaux1/build/MPI-prefix/src/MPI-stamp/MPI-configure-*.log

[ 43%] Performing build step for 'MPI'
-- MPI build command succeeded.  See also /Users/doutriaux1/build/MPI-prefix/src/MPI-stamp/MPI-build-*.log

[ 43%] Performing install step for 'MPI'
-- MPI install command succeeded.  See also /Users/doutriaux1/build/MPI-prefix/src/MPI-stamp/MPI-install-*.log

[ 43%] Completed 'MPI'
[ 43%] Built target MPI
Scanning dependencies of target ESMF
[ 43%] Creating directories for 'ESMF'
[ 44%] Performing download step (download, verify and extract) for 'ESMF'
-- downloading...
     src='http://uv-cdat.llnl.gov/cdat/resources/esmp.ESMF_6_3_0rp1_ESMP_01.tar.bz2'
     dst='/Users/doutriaux1/build/esmp.ESMF_6_3_0rp1_ESMP_01.tar.bz2'
     timeout='none'
attempt 1
-- [download 0% complete]
-- [download 100% complete]
-- downloading... done
-- verifying file...
     file='/Users/doutriaux1/build/esmp.ESMF_6_3_0rp1_ESMP_01.tar.bz2'
-- verifying file... done
-- extracting...
     src='/Users/doutriaux1/build/esmp.ESMF_6_3_0rp1_ESMP_01.tar.bz2'
     dst='/Users/doutriaux1/build/build/ESMF'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[ 44%] Performing patch step for 'ESMF'
patching file ESMP_LoadESMF.py
[ 44%] No update step for 'ESMF'
[ 45%] No configure step for 'ESMF'
[ 45%] Performing build step for 'ESMF'
CMake Error at /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build.cmake:56 (message):
  CTEST_FULL_OUTPUT

   Command failed: 1
   '/Applications/CMake.app/Contents/bin/cmake' '-P' '/Users/doutriaux1/build/CMake/ESMF_make_step.cmake'
   From /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build-out.log

  -------------------------------------------------

  -------------------------------------------------

   From /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build-err.log

  -------------------------------------------------

  CMake Warning (dev) in /Users/doutriaux1/build/CMake/ESMF_make_step.cmake:

    Syntax Warning in cmake code at

  

      /Users/doutriaux1/build/CMake/ESMF_make_step.cmake:31:28

  

    Argument not separated from preceding token by whitespace.

  This warning is for project developers.  Use -Wno-dev to suppress it.

  

  Make Errors detected:

   

  --------------------------------------------------------------

  Make version:

  GNU Make 3.81

  Copyright (C) 2006 Free Software Foundation, Inc.

  This is free software; see the source for copying conditions.

  There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A

  PARTICULAR PURPOSE.

  

  This program built for i386-apple-darwin11.3.0

  

  --------------------------------------------------------------

  Fortran Compiler version:

  

  --------------------------------------------------------------

  C++ Compiler version:

  

  --------------------------------------------------------------

  Preprocessor version:

  Apple LLVM version 5.1 (clang-503.0.40) (based on LLVM 3.4svn)

  Target: x86_64-apple-darwin12.5.0

  Thread model: posix

  

  --------------------------------------------------------------

  ESMF_VERSION_STRING: 6.3.0rp1

  --------------------------------------------------------------

   

  --------------------------------------------------------------

   * User set ESMF environment variables *

  ESMF_OS=Darwin

  ESMF_OPENMP=OFF

  ESMF_PTHREADS=OFF

  ESMF_DIR=/Users/doutriaux1/build/build/ESMF/esmf

  ESMF_ARRAYLITE=TRUE

  ESMF_COMM=openmpi

  ESMF_INSTALL_PREFIX=/Users/doutriaux1/build/install/Externals

  ESMF_MOAB=OFF

  ESMF_ABI=64

  ESMF_COMPILER=gfortran

   

  --------------------------------------------------------------

   * ESMF environment variables *

  ESMF_DIR: /Users/doutriaux1/build/build/ESMF/esmf

  ESMF_OS: Darwin

  ESMF_MACHINE: x86_64

  ESMF_ABI: 64

  ESMF_COMPILER: gfortran

  ESMF_BOPT: O

  ESMF_COMM: openmpi

  ESMF_SITE: default

  ESMF_PTHREADS: OFF

  ESMF_OPENMP: OFF

  ESMF_OPENACC: OFF

  ESMF_ARRAY_LITE: FALSE

  ESMF_NO_INTEGER_1_BYTE: TRUE

  ESMF_NO_INTEGER_2_BYTE: TRUE

  ESMF_FORTRANSYMBOLS: default

  ESMF_DEFER_LIB_BUILD: ON

  ESMF_SHARED_LIB_BUILD: ON

  ESMF_TESTEXHAUSTIVE: OFF

  ESMF_TESTWITHTHREADS: OFF

  ESMF_TESTMPMD: OFF

  ESMF_TESTSHAREDOBJ: OFF

  ESMF_TESTFORCEOPENMP: OFF

  ESMF_TESTFORCEOPENACC: OFF

  ESMF_TESTHARNESS_ARRAY: default

  ESMF_TESTHARNESS_FIELD: default

  ESMF_MPIRUN: mpirun

   

  --------------------------------------------------------------

   * ESMF environment variables pointing to 3rd party software *

  ESMF_MOAB: OFF

  ESMF_LAPACK: internal

   

  --------------------------------------------------------------

   * ESMF environment variables for final installation *

  ESMF_INSTALL_PREFIX: /Users/doutriaux1/build/install/Externals

  ESMF_INSTALL_HEADERDIR: include

  ESMF_INSTALL_MODDIR: mod/modO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_LIBDIR: lib/libO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_BINDIR: bin/binO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_DOCDIR: doc

   

  --------------------------------------------------------------

   * Compilers, Linkers, Flags, and Libraries *

  Location of the preprocessor: /usr/bin/gcc

  Location of the Fortran compiler:
  /Users/doutriaux1/build/install/Externals/bin/mpif90

  Location of the Fortran linker:
  /Users/doutriaux1/build/install/Externals/bin/mpif90

  Location of the C++ compiler:
  /Users/doutriaux1/build/install/Externals/bin/mpicxx

  Location of the C++ linker:
  /Users/doutriaux1/build/install/Externals/bin/mpicxx

  

  Fortran compiler flags:

  ESMF_F90COMPILEOPTS: -O -m64 -mcmodel=small -ffree-line-length-none

  ESMF_F90COMPILEPATHS:
  -I/Users/doutriaux1/build/build/ESMF/esmf/mod/modO/Darwin.gfortran.64.openmpi.default
  -I/Users/doutriaux1/build/build/ESMF/esmf/src/include

  ESMF_F90COMPILECPPFLAGS: -DESMF_NO_INTEGER_1_BYTE -DESMF_NO_INTEGER_2_BYTE
  -DESMF_LAPACK=1 -DESMF_MPIIO -DESMF_NO_PTHREADS -DESMF_NO_OPENMP
  -DESMF_NO_OPENACC -DSx86_64_small=1 -DESMF_OS_Darwin=1

  ESMF_F90COMPILEFREECPP:

  ESMF_F90COMPILEFREENOCPP: -ffree-form

  ESMF_F90COMPILEFIXCPP: -cpp -ffixed-form

  ESMF_F90COMPILEFIXNOCPP:

  

  Fortran linker flags:

  ESMF_F90LINKOPTS: -m64 -mcmodel=small

  ESMF_F90LINKPATHS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L

  ESMF_F90LINKRPATHS:

  ESMF_F90LINKLIBS: -lmpi_cxx -lstdc++

  ESMF_F90ESMFLINKLIBS: -lesmf -lmpi_cxx -lstdc++

  

  C++ compiler flags:

  ESMF_CXXCOMPILEOPTS: -O -DNDEBUG -DESMF_LOWERCASE_SINGLEUNDERSCORE -m64
  -mcmodel=small

  ESMF_CXXCOMPILEPATHS: -I/Users/doutriaux1/build/build/ESMF/esmf/src/include


  ESMF_CXXCOMPILECPPFLAGS: -DESMF_NO_INTEGER_1_BYTE -DESMF_NO_INTEGER_2_BYTE
  -DESMF_LAPACK=1 -DESMF_MPIIO -DESMF_NO_PTHREADS -DESMF_NO_OPENMP
  -DESMF_NO_OPENACC -DSx86_64_small=1 -DESMF_OS_Darwin=1 -D__SDIR__=''
  -DESMF_NO_SIGUSR2

  

  C++ linker flags:

  ESMF_CXXLINKOPTS: -m64 -mcmodel=small

  ESMF_CXXLINKPATHS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L

  ESMF_CXXLINKRPATHS:

  ESMF_CXXLINKLIBS: -lmpi_f77 -lgfortran

  ESMF_CXXESMFLINKLIBS: -lesmf -lmpi_f77 -lgfortran

  

  Shared library build:

  ESMF_SL_LIBS_TO_MAKE: libesmf

  ESMF_SL_SUFFIX: dylib

  ESMF_SL_LIBLINKER: mpicxx

  ESMF_SL_LIBOPTS: -dynamiclib

  ESMF_SL_LIBLIBS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L -lmpi_cxx -lstdc++
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L -lmpi_f77 -lgfortran

  

  

  --------------------------------------------------------------

  Compiling on Thu Feb 5 12:13:25 PST 2015 on sofia.llnl.gov

  Machine characteristics: Darwin sofia.llnl.gov 12.5.0 Darwin Kernel Version
  12.5.0: Sun Sep 29 13:33:47 PDT 2013; root:xnu-2050.48.12~1/RELEASE_X86_64
  x86_64

  ==============================================================

   

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  for library

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/mod/modO/Darwin.gfortran.64.openmpi.default
  for *.mod files

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/obj/objO/Darwin.gfortran.64.openmpi.default
  for *.o files

  Making directory /Users/doutriaux1/build/build/ESMF/esmf/src/include for
  include files

  Making directory /Users/doutriaux1/build/build/ESMF/esmf/src/etc for
  Attribute package files

  /Users/doutriaux1/build/build/ESMF/esmf/src

  tree_include tree_etc in: /Users/doutriaux1/build/build/ESMF/esmf/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue

  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness

  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs/mpiuni

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs/mpiuni

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/interface


  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/examples


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/examples


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/tests

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/examples


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/examples


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastru

  -------------------------------------------------



make[2]: *** [ESMF-prefix/src/ESMF-stamp/ESMF-build] Error 1
make[1]: *** [CMakeFiles/ESMF.dir/all] Error 2
make: *** [all] Error 2
doutriaux1@sofia:[~/build]:[9417]> 
```

**Comment by aashish24** [comment]
Looks like ESMF is broken. Can you confirm? If that' the case, we need to ping ESMF folks.

**Comment by doutriaux1** [comment]
it's not ESMF because -CDAT_BUILD_MODE=ALL works and that turns on parallel. So it's definitely our build.

**Comment by aashish24** [comment]
So basically in PARALLEL ON mode it seems like ParaView and ESMF both are failing.

**Comment by jbeezley** [comment]
The following worked for me at commit b7aada3 OS X 10.10.2 with
```
cmake -DCDAT_PARALLEL_BUILD=ON -DCDAT_BUILD_JPEG=OFF -DCDAT_USE_SYSTEM_JPEG=ON -DCDAT_BUILD_PNG=OFF -DCDAT_USE_SYSTEM_PNG=ON
```

Running ctest now, but they appear to be passing.

**Comment by doutriaux1** [comment]
yes but only in parallel=on build_mode=all works. must be dependencies order or missing

**Comment by doutriaux1** [comment]
@jbeezley i will try with these options

**Comment by doutriaux1** [comment]
well that's the issue no? -DCDAT_BUILD_PARRALEL=ON fails. Feel free to create another issue and close this one.

**Comment by aashish24** [comment]
Oh .. what you said confused me: "yes but only in parallel=on build_mode=all works. must be dependencies order or missing"

**Comment by doutriaux1** [comment]
sorry, I forgot the "dot" between the two options. BUILD_MODE=ALL does work (and it triggers PARALLEL=ON) so I think it is a dependencies order or dependencies missing.

**Comment by doutriaux1** [comment]
need to try on 10.9 too. Since it seems to work on 10.10 for @jbeezley

**Comment by aashish24** [comment]
@doutriaux1 can you try please?

**Comment by doutriaux1** [comment]
@aashish24 @jbeezley I get the exact same thing on 10.9

```
[ 77%] No update step for 'ESMF'
[ 78%] No configure step for 'ESMF'
[ 78%] Performing build step for 'ESMF'
-- PyQt configure command succeeded.  See also /Users/doutriaux1/build/PyQt-prefix/src/PyQt-stamp/PyQt-configure-*.log

[ 79%] Performing build step for 'PyQt'
remote: Total 328115 (delta 242890), reused 328115 (delta 242890)
Receiving objects: 100% (328115/328115), 120.18 MiB | 2.73 MiB/s, done.
Resolving deltas: 100% (242890/242890), done.
Checking connectivity... done.
Checking out files: 100% (6436/6436), done.
/Users/doutriaux1/build/build/ParaView
CMake Error at /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build.cmake:56 (message):
  CTEST_FULL_OUTPUT

   Command failed: 1
   '/Applications/CMake 2.8-12.app/Contents/bin/cmake' '-P' '/Users/doutriaux1/build/CMake/ESMF_make_step.cmake'
   From /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build-out.log

  -------------------------------------------------

  -------------------------------------------------

   From /Users/doutriaux1/build/ESMF-prefix/src/ESMF-stamp/ESMF-build-err.log

  -------------------------------------------------

  CMake Warning (dev) in /Users/doutriaux1/build/CMake/ESMF_make_step.cmake:

    Syntax Warning in cmake code at

  

      /Users/doutriaux1/build/CMake/ESMF_make_step.cmake:31:28

  

    Argument not separated from preceding token by whitespace.

  This warning is for project developers.  Use -Wno-dev to suppress it.

  

  Make Errors detected:

   

  --------------------------------------------------------------

  Make version:

  GNU Make 3.81

  Copyright (C) 2006 Free Software Foundation, Inc.

  This is free software; see the source for copying conditions.

  There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A

  PARTICULAR PURPOSE.

  

  This program built for i386-apple-darwin11.3.0

  

  --------------------------------------------------------------

  Fortran Compiler version:

  

  --------------------------------------------------------------

  C++ Compiler version:

  

  --------------------------------------------------------------

  Preprocessor version:

  Apple LLVM version 5.1 (clang-503.0.40) (based on LLVM 3.4svn)

  Target: x86_64-apple-darwin13.4.0

  Thread model: posix

  

  --------------------------------------------------------------

  ESMF_VERSION_STRING: 6.3.0rp1

  --------------------------------------------------------------

   

  --------------------------------------------------------------

   * User set ESMF environment variables *

  ESMF_OS=Darwin

  ESMF_OPENMP=OFF

  ESMF_PTHREADS=OFF

  ESMF_DIR=/Users/doutriaux1/build/build/ESMF/esmf

  ESMF_ARRAYLITE=TRUE

  ESMF_COMM=openmpi

  ESMF_INSTALL_PREFIX=/Users/doutriaux1/build/install/Externals

  ESMF_MOAB=OFF

  ESMF_ABI=64

  ESMF_COMPILER=gfortran

   

  --------------------------------------------------------------

   * ESMF environment variables *

  ESMF_DIR: /Users/doutriaux1/build/build/ESMF/esmf

  ESMF_OS: Darwin

  ESMF_MACHINE: x86_64

  ESMF_ABI: 64

  ESMF_COMPILER: gfortran

  ESMF_BOPT: O

  ESMF_COMM: openmpi

  ESMF_SITE: default

  ESMF_PTHREADS: OFF

  ESMF_OPENMP: OFF

  ESMF_OPENACC: OFF

  ESMF_ARRAY_LITE: FALSE

  ESMF_NO_INTEGER_1_BYTE: TRUE

  ESMF_NO_INTEGER_2_BYTE: TRUE

  ESMF_FORTRANSYMBOLS: default

  ESMF_DEFER_LIB_BUILD: ON

  ESMF_SHARED_LIB_BUILD: ON

  ESMF_TESTEXHAUSTIVE: OFF

  ESMF_TESTWITHTHREADS: OFF

  ESMF_TESTMPMD: OFF

  ESMF_TESTSHAREDOBJ: OFF

  ESMF_TESTFORCEOPENMP: OFF

  ESMF_TESTFORCEOPENACC: OFF

  ESMF_TESTHARNESS_ARRAY: default

  ESMF_TESTHARNESS_FIELD: default

  ESMF_MPIRUN: mpirun

   

  --------------------------------------------------------------

   * ESMF environment variables pointing to 3rd party software *

  ESMF_MOAB: OFF

  ESMF_LAPACK: internal

   

  --------------------------------------------------------------

   * ESMF environment variables for final installation *

  ESMF_INSTALL_PREFIX: /Users/doutriaux1/build/install/Externals

  ESMF_INSTALL_HEADERDIR: include

  ESMF_INSTALL_MODDIR: mod/modO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_LIBDIR: lib/libO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_BINDIR: bin/binO/Darwin.gfortran.64.openmpi.default

  ESMF_INSTALL_DOCDIR: doc

   

  --------------------------------------------------------------

   * Compilers, Linkers, Flags, and Libraries *

  Location of the preprocessor: /usr/bin/gcc

  Location of the Fortran compiler:
  /Users/doutriaux1/build/install/Externals/bin/mpif90

  Location of the Fortran linker:
  /Users/doutriaux1/build/install/Externals/bin/mpif90

  Location of the C++ compiler:
  /Users/doutriaux1/build/install/Externals/bin/mpicxx

  Location of the C++ linker:
  /Users/doutriaux1/build/install/Externals/bin/mpicxx

  

  Fortran compiler flags:

  ESMF_F90COMPILEOPTS: -O -m64 -mcmodel=small -ffree-line-length-none

  ESMF_F90COMPILEPATHS:
  -I/Users/doutriaux1/build/build/ESMF/esmf/mod/modO/Darwin.gfortran.64.openmpi.default
  -I/Users/doutriaux1/build/build/ESMF/esmf/src/include

  ESMF_F90COMPILECPPFLAGS: -DESMF_NO_INTEGER_1_BYTE -DESMF_NO_INTEGER_2_BYTE
  -DESMF_LAPACK=1 -DESMF_MPIIO -DESMF_NO_PTHREADS -DESMF_NO_OPENMP
  -DESMF_NO_OPENACC -DSx86_64_small=1 -DESMF_OS_Darwin=1

  ESMF_F90COMPILEFREECPP:

  ESMF_F90COMPILEFREENOCPP: -ffree-form

  ESMF_F90COMPILEFIXCPP: -cpp -ffixed-form

  ESMF_F90COMPILEFIXNOCPP:

  

  Fortran linker flags:

  ESMF_F90LINKOPTS: -m64 -mcmodel=small

  ESMF_F90LINKPATHS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L

  ESMF_F90LINKRPATHS:

  ESMF_F90LINKLIBS: -lmpi_cxx -lstdc++

  ESMF_F90ESMFLINKLIBS: -lesmf -lmpi_cxx -lstdc++

  

  C++ compiler flags:

  ESMF_CXXCOMPILEOPTS: -O -DNDEBUG -DESMF_LOWERCASE_SINGLEUNDERSCORE -m64
  -mcmodel=small

  ESMF_CXXCOMPILEPATHS: -I/Users/doutriaux1/build/build/ESMF/esmf/src/include


  ESMF_CXXCOMPILECPPFLAGS: -DESMF_NO_INTEGER_1_BYTE -DESMF_NO_INTEGER_2_BYTE
  -DESMF_LAPACK=1 -DESMF_MPIIO -DESMF_NO_PTHREADS -DESMF_NO_OPENMP
  -DESMF_NO_OPENACC -DSx86_64_small=1 -DESMF_OS_Darwin=1 -D__SDIR__=''
  -DESMF_NO_SIGUSR2

  

  C++ linker flags:

  ESMF_CXXLINKOPTS: -m64 -mcmodel=small

  ESMF_CXXLINKPATHS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L

  ESMF_CXXLINKRPATHS:

  ESMF_CXXLINKLIBS: -lmpi_f77 -lgfortran

  ESMF_CXXESMFLINKLIBS: -lesmf -lmpi_f77 -lgfortran

  

  Shared library build:

  ESMF_SL_LIBS_TO_MAKE: libesmf

  ESMF_SL_SUFFIX: dylib

  ESMF_SL_LIBLINKER: mpicxx

  ESMF_SL_LIBOPTS: -dynamiclib

  ESMF_SL_LIBLIBS:
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L -lmpi_cxx -lstdc++
  -L/Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  -L -lmpi_f77 -lgfortran

  

  

  --------------------------------------------------------------

  Compiling on Mon Feb 9 14:33:17 PST 2015 on maryam

  Machine characteristics: Darwin maryam 13.4.0 Darwin Kernel Version 13.4.0:
  Sun Aug 17 19:50:11 PDT 2014; root:xnu-2422.115.4~1/RELEASE_X86_64 x86_64

  ==============================================================

   

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/lib/libO/Darwin.gfortran.64.openmpi.default
  for library

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/mod/modO/Darwin.gfortran.64.openmpi.default
  for *.mod files

  Making directory
  /Users/doutriaux1/build/build/ESMF/esmf/obj/objO/Darwin.gfortran.64.openmpi.default
  for *.o files

  Making directory /Users/doutriaux1/build/build/ESMF/esmf/src/include for
  include files

  Making directory /Users/doutriaux1/build/build/ESMF/esmf/src/etc for
  Attribute package files

  /Users/doutriaux1/build/build/ESMF/esmf/src

  tree_include tree_etc in: /Users/doutriaux1/build/build/ESMF/esmf/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue

  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/prologue/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness

  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/test_harness/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs/mpiuni

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/stubs/mpiuni

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Util/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/LogErr/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/VM/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/src

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/interface

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/interface

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Base/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/interface


  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/examples


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/examples


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/TimeMgr/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/tests

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/examples


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/examples


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/DELayout/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/examples

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/examples

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/doc

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/doc

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/tests

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/Config/tests

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec

  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/src

  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/src

  
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/interface


  tree_include tree_etc in:
  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/interface


  /Users/doutriaux1/build/build/ESMF/esmf/src/Infrastructure/ArraySpec/

  -------------------------------------------------



make[2]: *** [ESMF-prefix/src/ESMF-stamp/ESMF-build] Error 1
make[1]: *** [CMakeFiles/ESMF.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
Branch uvcdat-master set up to track remote branch uvcdat-master from origin.
Switched to a new branch 'uvcdat-master'
Submodule 'ThirdParty/IceT/vtkicet' (http://paraview.org/IceT.git) registered for path 'ThirdParty/IceT/vtkicet'
Submodule 'ThirdParty/QtTesting/vtkqttesting' (http://paraview.org/QtTesting.git) registered for path 'ThirdParty/QtTesting/vtkqttesting'
Submodule 'ThirdParty/protobuf/vtkprotobuf' (http://public.kitware.com/protobuf.git) registered for path 'ThirdParty/protobuf/vtkprotobuf'
Submodule 'Utilities/VisItBridge' (http://paraview.org/VisItBridge.git) registered for path 'Utilities/VisItBridge'
Submodule 'VTK' (http://vtk.org/VTK.git) registered for path 'VTK'
Cloning into 'ThirdParty/IceT/vtkicet'...
remote: Counting objects: 4203, done.
remote: Compressing objects: 100% (1046/1046), done.
remote: Total 4203 (delta 3152), reused 4203 (delta 3152)
Receiving objects: 100% (4203/4203), 1.06 MiB | 1.03 MiB/s, done.
Resolving deltas: 100% (3152/3152), done.
Checking connectivity... done.
Submodule path 'ThirdParty/IceT/vtkicet': checked out 'c6166712c7e86a4cded4b2991e6e7cb9224a3b6e'
Cloning into 'ThirdParty/QtTesting/vtkqttesting'...
remote: Counting objects: 1536, done.
remote: Compressing objects: 100% (385/385), done.
remote: Total 1536 (delta 1149), reused 1536 (delta 1149)
Receiving objects: 100% (1536/1536), 389.22 KiB | 589.00 KiB/s, done.
Resolving deltas: 100% (1149/1149), done.
Checking connectivity... done.
Submodule path 'ThirdParty/QtTesting/vtkqttesting': checked out '2a4311004c02905e05eca1b07bf444c4c140fb12'
Cloning into 'ThirdParty/protobuf/vtkprotobuf'...
remote: Counting objects: 815, done.
remote: Compressing objects: 100% (437/437), done.
remote: Total 815 (delta 350), reused 815 (delta 350)
Receiving objects: 100% (815/815), 1.80 MiB | 1.49 MiB/s, done.
Resolving deltas: 100% (350/350), done.
Checking connectivity... done.
Submodule path 'ThirdParty/protobuf/vtkprotobuf': checked out 'ef5922053d03773f12212de5285a436b2a23c01b'
Cloning into 'Utilities/VisItBridge'...
remote: Counting objects: 9202, done.
remote: Compressing objects: 100% (2515/2515), done.
remote: Total 9202 (delta 6672), reused 9202 (delta 6672)
Receiving objects: 100% (9202/9202), 9.63 MiB | 2.87 MiB/s, done.
Resolving deltas: 100% (6672/6672), done.
Checking connectivity... done.
Submodule path 'Utilities/VisItBridge': checked out 'f0f2a17e9cf41c7b9472f30a63b22671d24dcb9f'
Cloning into 'VTK'...
remote: Counting objects: 396110, done.
remote: Compressing objects: 100% (87089/87089), done.
remote: Total 396110 (delta 308821), reused 391251 (delta 304017)
Receiving objects: 100% (396110/396110), 112.64 MiB | 783.00 KiB/s, done.
Resolving deltas: 100% (308821/308821), done.
Checking connectivity... done.
Submodule path 'VTK': checked out '6bd6506a2013c0c70410f3fb0bb3812e71fe6ab0'
[ 80%] No patch step for 'ParaView'
[ 80%] No update step for 'ParaView'
[ 80%] Performing configure step for 'ParaView'
-- ParaView configure command succeeded.  See also /Users/doutriaux1/build/ParaView-prefix/src/ParaView-stamp/ParaView-configure-*.log

[ 80%] Performing build step for 'ParaView'
CMake Error at /Users/doutriaux1/build/ParaView-prefix/src/ParaView-stamp/ParaView-build.cmake:56 (message):
  CTEST_FULL_OUTPUT

   Command failed: 2
   '/Applications/Xcode.app/Contents/Developer/usr/bin/make'
   From /Users/doutriaux1/build/ParaView-prefix/src/ParaView-stamp/ParaView-build-out.log

  -------------------------------------------------

  Scanning dependencies of target ParaViewData

  [ 0%] Built target ParaViewData

  Scanning dependencies of target vtkWrappingTools

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParse.tab.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseData.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseMain.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseExtras.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseMangle.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseString.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParseHierarchy.c.o

  Scanning dependencies of target vtksys

  [ 0%] Building C object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/ProcessUNIX.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkParsePreprocess.c.o

  [ 0%] Building C object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/Base64.c.o

  [ 0%] [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkWrap.c.o

  Building C object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/EncodingC.c.o

  [ 0%] Building C object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/MD5.c.o

  [ 0%] Building C object
  VTK/Wrapping/Tools/CMakeFiles/vtkWrappingTools.dir/vtkWrapText.c.o

  [ 0%] Building C object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/System.c.o

  Linking C static library ../../../lib/libvtkWrappingTools-pv4.1.a

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/Directory.cxx.o

  [ 0%] Built target vtkWrappingTools

  Scanning dependencies of target vtkalglib

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/ap.cpp.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/DynamicLoader.cxx.o

  [ 0%] [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/EncodingCXX.cxx.o

  Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/bdsvd.cpp.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/bidiagonal.cpp.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/Glob.cxx.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/blas.cpp.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/lq.cpp.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/RegularExpression.cxx.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/qr.cpp.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/SystemTools.cxx.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/reflections.cpp.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/rotations.cpp.o

  [ 0%] Building CXX object
  VTK/ThirdParty/alglib/CMakeFiles/vtkalglib.dir/svd.cpp.o

  Linking CXX shared library ../../../lib/libvtkalglib-pv4.1.dylib

  [ 0%] Built target vtkalglib

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/CommandLineArguments.cxx.o


  Scanning dependencies of target vtkDICOMParser

  [ 0%] Building CXX object
  VTK/Utilities/DICOMParser/CMakeFiles/vtkDICOMParser.dir/DICOMFile.cxx.o

  [ 0%] [ 0%] Building CXX object
  VTK/Utilities/DICOMParser/CMakeFiles/vtkDICOMParser.dir/DICOMParser.cxx.o

  Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/FStream.cxx.o

  [ 0%] Building CXX object
  VTK/Utilities/KWSys/vtksys/CMakeFiles/vtksys.dir/SystemInformation.cxx.o

  [ 0%] Building CXX object
  VTK/Utilities/DICOMParser/CMakeFiles/vtkDICOMParser.dir/DICOMAppHelper.cxx.o


  Linking CXX shared library ../../../../lib/libvtksys-pv4.1.dylib

  [ 0%] Built target vtksys

  Scanning dependencies of target vtkmetaio

  [ 0%] Building CXX object
  VTK/Utilities/MetaIO/vtkmetaio/CMakeFiles/vtkmetaio.dir/metaUtils.cxx.o

  Linking CXX shared library ../../../lib/libvtkDICOMParser-pv4.1.dylib

  [ 0%] Built target vtkDICOMParser

  [ 0%] Building CXX object
  VTK/Utilities/MetaIO/vtkmetaio/CMakeFiles/vtkmetaio.dir/metaArrow.cxx.o

  Scanning dependencies of target vtkpng

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngget.c.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngrio.c.o

  [ 0%] Building CXX object
  VTK/Utilities/MetaIO/vtkmetaio/CMakeFiles/vtkmetaio.dir/metaBlob.cxx.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngwrite.c.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/png.c.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngmem.c.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngrtran.c.o

  [ 0%] Building C object
  VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/pngtrans.c.o

  [ 0%] Building C object VTK/ThirdParty/png/vtkpng/CMakeFiles/vtkpng.dir/

  -------------------------------------------------

   From /Users/doutriaux1/build/ParaView-prefix/src/ParaView-stamp/ParaView-build-err.log

  -------------------------------------------------

  
  /Users/doutriaux1/build/build/ParaView/ThirdParty/IceT/vtkicet/src/strategies/radixk.c:24:10:
  fatal error: 'mpi.h' file not found

  #include <mpi.h>

           ^

  1 error generated.

  make[5]: ***
  [ThirdParty/IceT/vtkicet/src/ice-t/CMakeFiles/IceTCore.dir/__/strategies/radixk.c.o]
  Error 1

  make[4]: ***
  [ThirdParty/IceT/vtkicet/src/ice-t/CMakeFiles/IceTCore.dir/all] Error 2

  make[4]: *** Waiting for unfinished jobs....

  make[3]: *** [all] Error 2

  -------------------------------------------------



make[2]: *** [ParaView-prefix/src/ParaView-stamp/ParaView-build] Error 1
make[1]: *** [CMakeFiles/ParaView.dir/all] Error 2
```

**Comment by dlonie** [comment]
Closing, we don't have ParaView in the superbuild anymore.
